app/controllers/dashboards_controller.rb

```diff
@@ -11,6 +11,7 @@ def search
     end
     @patient = Patient.new
     @pregnancy = @patient.pregnancies.new
+    @date=Time.now.to_date
```

Comment:
@Kevin-Wei I think the current Rubocop standard is `Time.zone.today`. I'm inclined to go with that, what do you think @Nico-ehs ?